Extensions/PlayerAuthentication/playerauthenticationtools.ts

```diff
@@ -362,15 +423,42 @@ namespace gdjs {
     };
 
     /**
-     * Clear the authentication window timeout.
+     * Clear all existing authentication timeouts.
      * Useful when:
+     * - a new authentication starts
      * - the authentication succeeded
      * - the authentication window is closed
      */
     const clearAuthenticationWindowTimeout = () => {
+      if (_initialAuthenticationTimeoutId)
+        clearTimeout(_initialAuthenticationTimeoutId);
       if (_authenticationTimeoutId) clearTimeout(_authenticationTimeoutId);
     };
 
+    /**
+     * Helper to create the authentication banner based on the authentication status.
+     */
+    const createAuthenticationBanner = function (
+      runtimeScene: gdjs.RuntimeScene
+    ) {
```

Comment:
Nit: can you type the return type (`HTMLDivElement `) so that it's easier to understand what is created?